cms/templates/js/login.underscore

```diff
@@ -0,0 +1,12 @@
+<div class="wrapper-content wrapper">
+    <form id="login_form" method="post" action="login_post" onsubmit="return false;">
+        <input type="hidden" name="csrfmiddlewaretoken" value="csrf"/>
+        <input id="email" type="email" name="email" placeholder="'example: username@domain.com'"/>
+        <input id="password" type="password" name="password"/>
+
+        <div class="form-actions">
+            <button type="submit" id="submit" name="submit" class="action action-primary">Sign In</button>
```

Comment:
Can we do something like this just to keep Internationalization and localization in mind?
`gettext("Sign in")`

What do you think?